app/code/Magento/Config/Model/Config/Source/Nooptreq.php

```diff
@@ -11,15 +11,19 @@
  */
 class Nooptreq implements \Magento\Framework\Option\ArrayInterface
 {
+    const VALUE_NO = '';
```

Comment:
Sadly while putting this to the `2.2-develop` branch and with this class marked as `@api` we cannot add any new constants. For a PR to `2.3-develop` we could add these new constants. For this PR we will have to change this back.

Reply:
Can you explain why adding new constants as an issue? It doesn't affect backwards compatibility and only improves things, I don't see any disadavantage of that... sounds like a quite arbitrary rule to me, but perhaps it gets clearer if you explain.

Reply:
Hi @dmanners in general, you are right. But in this particular, we can accept new constants. There are some cases when new constants can break client code, but the chances a very low.
@dmanners @avstudnitz thank you for collaboration